### PR TITLE
Adding id and name for Input

### PR DESCRIPTION
**Pull request by girishdudhe**
name filed in missing in Input component and id is not correctly assigned.So corrected the code to add id and name in Input.

### CODE REVIEW

File: src/forms/input/Input.tsx [hunk 1]
```diff
@@ -61,46 +63,58 @@ export class Input extends React.PureComponent<InputProps> {
...
+                            id={setId ? setId : id}
+                            name={name}
                             value={value}
                             defaultValue={defaultValue}
                             size={size}
                             disabled={disabled}
-                            className="clr-input"
                             placeholder={placeholder}
                             onChange={this.handleChange}
+                            className={className}
```
Comment (akate1) [inline src/forms/input/Input.tsx]: I think we also need to keep class name "clr-input" and then add user define class names after that.

File: src/forms/input/Input.tsx [hunk 2]
```diff
@@ -61,46 +63,58 @@ export class Input extends React.PureComponent<InputProps> {
...
+                            <div className="clr-input-wrapper">
+                                <input
+                                    type={type || "text"}
+                                    name={name}
+                                    id={setId ? setId : id}
+                                    value={value}
+                                    defaultValue={defaultValue}
+                                    size={size}
+                                    disabled={disabled}
+                                    className="clr-input"
```
Comment (akate1) [inline src/forms/input/Input.tsx]: Same here keep class name 'clr-input' and append user defined class names after that

File: src/forms/input/Input.tsx (hunk 1 above)
Comment (girishdudhe) [reply src/forms/input/Input.tsx]: I have not changes .... so dropping the comment

File: src/forms/input/Input.tsx (hunk 2 above)
Comment (girishdudhe) [reply src/forms/input/Input.tsx]: dropping comment as per above

**Pull request merged**